web/src/main/java/org/cbioportal/web/SessionServiceController.java

```diff
@@ -50,6 +51,15 @@ public class SessionServiceController {
     @Value("${session.service.url:}")
     private String sessionServiceURL;
 
+    private static Map<SessionPage, Class<? extends PageSettingsData>> pageTypeToData;
+    
+    static {
+         SessionServiceController.pageTypeToData = ImmutableMap.of(
+             SessionPage.result_view, ResultPageSettings.class,
+             SessionPage.study_view, StudyPageSettings.class
+         );
+    }
+    
```

Comment:
Why are we splitting declaration and initialization?

Reply:
Yes, I removed this split. Agree @BasLee ?

Reply:
I agree, solved in [e15f7c6](https://github.com/cBioPortal/cbioportal/pull/9450/commits/e15f7c68dc730c3259961ce9cf32af1539a7dc5d)
I think we can merge?